### DISCUSSION TITLE
Not having the full EmguCv runtime

### DISCUSSION THREAD

**Discussion by Lethargie1** [emgucv/emgucv#929] [Q&A]
Hello, I'm starting to use emgucv. Since I plan on using both 3d mesh with corresponding image I decided to try to make a simple viz3d window to display basic stuff. I get a strange error message though:
`The library is compiled without Viz support. To use this module, please switch to the full Emgu CV runtime.`

I can't figure out what is the "full Emgu CV runtime". I tried to recompile the whole project from the git repository, but I get the same error

**Comment by canmingh** [comment]
On what platform are you using Emgu CV? Do you use nuget package? If so, which package?

**Reply by Lethargie1** [reply]
This is on a normal windows platform, I used the nuget, manual download and compiling from the git source. The 3 options led to the same error message.

using nuget it was
dotnet add package Emgu.CV.runtime.windows --version 4.8.1.5350
and 
dotnet add package Emgu.CV --version 4.8.1.5350

I ran the hello world example with no problem
`Viz3d Window3d = new Viz3d("Vizualizer");
`
is what caused the error

**Comment by emgucv** [comment]
The Viz3d model requried the Open CV binary to be build with VTK. On windows, only the commercial release is built with VTK. The  Emgu.CV.runtime.windows.dldt nuget package from the commercial nuget repository has this module enabled. The regular open source release from nuget.org do not have this module enabled.
